rotkehlchen/tests/unit/test_serialization.py

```diff
@@ -97,3 +108,40 @@ def test_deserialize_int_from_hex_or_int():
     assert deserialize_int_from_hex_or_int('0x1', 'whatever') == 1
     assert deserialize_int_from_hex_or_int('0x33', 'whatever') == 51
     assert deserialize_int_from_hex_or_int(66, 'whatever') == 66
+
+
+def test_deserialize_ethereum_transaction():
+    # Sometimes we can get transactions where "to address" is None
```

Comment:
This can happen yes. An ethereum transaction with a `to` missing means that the `from` address is deploying a new contract.

I would suggest that in this test test you test this kind of transaction. So rename it to `test_deserialize_deployment_ethereum_transaction` and remove the comments, or add it as a docstring.

In the ethereum world it's well known that empty `to` should mean contract deployment.